src/site/content/en/blog/declarative-link-capturing/index.md

```diff
@@ -0,0 +1,223 @@
+
+To experiment with Declarative Link Capturing locally, without an origin trial token, enable the
+`#enable-desktop-pwas-link-capturing ` flag in `about://flags`.
+
+### Enabling support during the origin trial phase
+
+Starting in Chromium&nbsp;91, Declarative Link Capturing will be available as an origin trial in
+Chromium. The origin trial is expected to end in Chromium&nbsp;94 (October 13, 2021).
+
+{% Aside 'caution' %} The origin trial will initially be limited to the Chrome&nbsp;OS platform.
+{% endAside %}
+
+{% include 'content/origin-trials.njk' %}
+
+### Register for the origin trial {: #register-for-ot }
+
+{% include 'content/origin-trial-register.njk' %}
+
+## How to use Declarative Link Capturing? {: #use }
+
+Developers can declaratively determine how links should be captured by leveraging the additional Web
+Application Manifest field `"capture_links"`. It takes a string or a an array of strings as its
+value. If an array of strings is given, the user agent chooses the first supported item in the list,
+defaulting to `"none"`. The possible values dictate the link capturing behavior as follows:
+
+- `"none"` (the default): No link capturing; links clicked leading to this PWA scope navigate as
+  normal without opening a PWA window.
+- `"new-client"`: Each link clicked leading to this PWA scope opens a new PWA window at that URL.
+- `"existing-client-navigate"`: When a link is clicked leading to this PWA scope, the user agent
+  finds an existing PWA window, and causes it to navigate to the opened URL. If more than one PWA
+  window exists, the browser may choose one arbitrarily. This behaves like `"new-client"` if no
+  window is currently open. 🚨 Careful! This option potentially leads to data loss, as pages can be
+  arbitrarily navigated away from. Sites should be aware that they are opting into such behavior by
+  choosing this option. This option works best for "read-only" sites that do not hold user data in
+  memory, such as music players. If the page being navigated away from has a
+  [`beforeunload` event](https://developer.mozilla.org/en-US/docs/Web/API/WindowEventHandlers/onbeforeunload),
+  the user would see the prompt before the navigation completes.
+- `"existing-client-event"`: When a link is clicked leading to this PWA scope the user agent finds
+  an existing PWA window and fires a `launch` event in that window's top-level context, containing
+  the launched URL. If more than one PWA window exists, the browser may choose one arbitrarily. This
+  behaves as `"new-client"` if no window is currently open. This is intended for more advanced
+  sites, which inspect the launched URL and use it to load data into the current browsing context,
+  without a navigation. For example, a site that allows multiple documents to be loaded in the same
+  window can pop open a new sub-window in response to the navigation.
+
+{% Aside 'objective' %} There is ongoing discussion about adding a `"serviceworker"` option in the
```

Comment:
Put `existing-client-event` here too. So you could say "an ongoing discussion about several new options in the future that do not open a window at all, but instead fire a `launch` event in a chosen foreground window, or the service worker."

Reply:
'Objective' is for the objective of a less, not the objective of a product or project.

Reply:
Removed the objective and used Matt's phrase.